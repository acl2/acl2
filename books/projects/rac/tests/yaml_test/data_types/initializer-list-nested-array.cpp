#include <array>
using namespace std;

// RAC begin

int bar() {
  const array<int, 2> arr[2] = { 3 }, arr2[1] = { 0 }, arr3[1] = { 0 };
  return 0;
}
