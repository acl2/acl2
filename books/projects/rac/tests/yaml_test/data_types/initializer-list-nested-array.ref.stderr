initializer-list-nested-array.cpp:7 (22-25): Wrong type provided, expected int[2] but got int
  7 |   const array<int, 2> arr[2] = { 3 }, arr2[1] = { 0 }, arr3[1] = { 0 };
    |                       ^^^
